menpo/io/input/video.py

```diff
@@ -1,55 +1,43 @@
 import warnings
-from functools import partial
+import os
+import numpy as np
+import subprocess as sp
+import re
+from pathlib import Path
 
 from menpo.image.base import normalize_pixels_range, channels_to_front
 from menpo.image import Image
 from menpo.base import LazyList
 
+from ..utils import DEVNULL, _call_subprocess
 
-def _imageio_to_menpo(imio_reader, normalize, index):
-    pixels = imio_reader.get_data(index)
-    pixels = channels_to_front(pixels)
 
-    if normalize:
-        return Image(normalize_pixels_range(pixels), copy=False)
-    else:
-        return Image(pixels, copy=False)
-
-
-def ffmpeg_types():
-    r"""The supported FFMPEG types.
-
-    Returns
-    -------
-    supported_types : `dict`
-        A dictionary of extensions to the :map:`imageio_ffmpeg_importer`.
-    """
-    try:
-        import imageio
-
-        # Lazy way to get all the extensions supported by imageio/FFMPEG
-        ffmpeg_exts = imageio.formats['ffmpeg'].extensions
-        return dict(zip(ffmpeg_exts,
-                        [imageio_ffmpeg_importer] * len(ffmpeg_exts)))
-    except ImportError:
-        return {}
+_FFMPEG_CMD = lambda: str(Path(os.environ.get('MENPO_FFMPEG_CMD', 'ffmpeg')))
```

Comment:
Am I misreading this, or does this now require that `MENPO_FFMPEG_CMD` must be set for video IO to work? I figured it would be a fallback to customise?